snappy_pipeline/workflows/somatic_variant_filtration/__init__.py

```diff
@@ -155,6 +155,7 @@
     tools_somatic_variant_calling: null                 # Default: use those defined in somatic_variant_calling step
     tools_somatic_variant_annotation: null              # Default: use those defined in somatic_variant_annotation step
     has_annotation: True
+    filtration_schema: "sets"                           # Either "sets" (old scheme- filter_sets) or "list" (new scheme- filter_list)
```

Comment:
Default is changed. `sets` is because of `filter_sets`, I didn't want to break the config unnecessarily. But I'll change it if you feel it's important.